config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_api_trace_attributes.conf

```diff
@@ -226,6 +226,15 @@ commands = [
               scope: API_TRACE,
               sources: [QS],
               type: ATTRIBUTE
+            },
+            {
+              fqn: Api.Trace.api_trace_error_span_count,
+              key: apiTraceErrorSpanCount,
+              value_kind: TYPE_INT64,
+              display_name: Api Trace Error Span Count,
```

Comment:
We usually not prefix Api Trace, since it's already under the API_TRACE scope, unless we really want to display it like this in the UI, which I doubt